src/include/rbd/librbd.h

```diff
@@ -1526,7 +1526,7 @@ CEPH_RBD_API int rbd_quiesce_watch(rbd_image_t image,
  * @param handle which watch is complete
  * @param r the return code
  */
-CEPH_RADOS_API void rbd_quiesce_complete(rbd_image_t image, uint64_t handle,
+CEPH_RBD_API void rbd_quiesce_complete(rbd_image_t image, uint64_t handle,
                                          int r);
```

Comment:
Please adjust formatting:
```suggestion
                                       int r);

```

Reply:
thanx